Add 5-year anniversary profile illustration to handbook (#16964)
PostHog/posthog.com: contents/handbook/people/time-off.md

3. Rimowa luggage set (large trunk, cabin bag, packing cube, toiletries bag)

On the run up to your anniversary, our Ops team will send you a link to the gift options questionnaire and order your 4 year anniversary gift once we receive your completed form. Thank you for making PostHog great!

#### 5th year anniversary

> To mark five years at PostHog, you can get a brand new hedgehog profile illustration drawn to replace the one taken when you joined. Send an updated photo to our Ops team and they'll get it over to our contract illustrator. Once it's ready, the new illustration will be swapped in on your team profile.

Author: ```suggestion
To mark five years at PostHog, you can get a brand new hedgehog profile illustration drawn to replace the one taken when you joined. Send an updated photo to our Graphics team and they'll get it over to our illustrator. Once it's ready, the new illustration will be sent to you.
```
